_artifacts/document-attributes.adoc

```diff
@@ -97,20 +97,20 @@ endif::OP[]
 :CODEREADY_STUDIO_VERSION: 12.17
 :CODEREADY_STUDIO_VERSION_LONG: 12.17.0
 :DATAGRID: Red Hat Data Grid
-:DATAGRID_VERSION: 7.3
-:DATAGRID_VERSION_LONG: 7.3.6
+:DATAGRID_VERSION: 8.0
+:DATAGRID_VERSION_LONG: 8.0.0
 :EAP: Red Hat JBoss EAP
 :EAP_LONG: Red Hat JBoss Enterprise Application Platform
-:EAP_VERSION: 7.3
-:EAP_VERSION_LONG: 7.3.0
+:EAP_VERSION: 7.4
+:EAP_VERSION_LONG: 7.4.0
 :FUSE: Fuse
 :FUSE_LONG: Red Hat Fuse
 :FUSE_VERSION: 7.9
 :FUSE_EAP_VERSION: 7.4
 :SPRING_BOOT_VERSION: 2.3.4
 :JWS: Red Hat JBoss Web Server
-:JWS_VERSION: 5.4
-:JWS_VERSION_LONG: 5.4.0
+:JWS_VERSION: 5.5
+:JWS_VERSION_LONG: 5.5.0
```

Comment:
@sutaakar Are these versions correct?

Reply:
there is a 5.5.1 version already available, we may upgrade to it.

Reply:
> there is a 5.5.1 version already available, we may upgrade to it.

I'm afraid I'm going to need a something a bit more concrete, guys. ;o) Should I keep it at 5.5.0 or change to 5.5.1?

Reply:
@briandooley Please change it to 5.5.1.